### ISSUE TITLE
The `lint-audit` step in CircleCI doesn't appear to do anything

### ISSUE THREAD

**Issue by toufali** [mozilla/blurts-server#2344]
The `lint-audit` step uses the 3rd party dependency `npm-audit-ci-wrapper`, which is deprecated.  It seems like this step is no-op, since there have been numerous audit alerts that have gone un-flagged.

We should refactor the step, possibly using npm-native audit tools.  Alternatively we can likely remove the step altogether, given that GitHub provides audits as well.

In conjunction with the audit step, we might also run a dependency check, possibly via `depcheck`.  See #2316

**Comment by pdehaan** [comment]
Actually, ignore that last part... Per the linked 179 issue:
> This issue has been resolved in version 4.0.9

Yet, the `@beta` dist-tag is somewhat old (pre-v4.0.1 vs `@latest`'s 4.0.32; not sure where I grabbed that command that suggested using beta release via npx):

```sh
npm info auditjs dist-tags

{
  beta: '4.0.1-beta.33',
  latest: '4.0.32',
  beta1: '2.0.0',
  alpha: '4.0.1-alpha.38'
}
```

So this updated command gives me the sauce I want:

```sh
npx auditjs ossi --quiet --json
npx: installed 105 in 5.1s
[
  {
    "coordinates": "pkg:npm/glob-parent@3.1.0",
    "description": "Extract the non-magic parent path from a glob string.",
    "reference": "https://ossindex.sonatype.org/component/pkg:npm/glob-parent@3.1.0?utm_source=auditjs&utm_medium=integration&utm_content=4.0.1-beta.33",
    "vulnerabilities": [
      {
        "id": "64cd5f21-8af4-4eae-ac7d-a53241ea693a",
        "title": "CWE-400: Uncontrolled Resource Consumption ('Resource Exhaustion')",
        "description": "The software does not properly restrict the size or amount of resources that are requested or influenced by an actor, which can be used to consume more resources than intended.",
        "cvssScore": 7.5,
        "cvssVector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
        "reference": "https://ossindex.sonatype.org/vulnerability/64cd5f21-8af4-4eae-ac7d-a53241ea693a?component-type=npm&component-name=glob-parent&utm_source=auditjs&utm_medium=integration&utm_content=4.0.1-beta.33"
      }
    ]
  },
  {
    "coordinates": "pkg:npm/yargs-parser@5.0.1",
    "description": "the mighty option parser used by yargs",
    "reference": "https://ossindex.sonatype.org/component/pkg:npm/yargs-parser@5.0.1?utm_source=auditjs&utm_medium=integration&utm_content=4.0.1-beta.33",
    "vulnerabilities": [
      {
        "id": "b7740d41-fc85-4d22-8af5-5a3159e114ea",
        "title": "[CVE-2020-7608] yargs-parser could be tricked into adding or modifying properties of Object.prot...",
        "description": "yargs-parser could be tricked into adding or modifying properties of Object.prototype using a \"__proto__\" payload.",
        "cvssScore": 7.5,
        "cvssVector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:H/A:N",
        "cve": "CVE-2020-7608",
        "reference": "https://ossindex.sonatype.org/vulnerability/b7740d41-fc85-4d22-8af5-5a3159e114ea?component-type=npm&component-name=yargs-parser&utm_source=auditjs&utm_medium=integration&utm_content=4.0.1-beta.33"
      },
      {
        "id": "7ccaaed0-205b-4382-a963-8a30a0b151b1",
        "title": "CWE-400: Uncontrolled Resource Consumption ('Resource Exhaustion')",
        "description": "The software does not properly restrict the size or amount of resources that are requested or influenced by an actor, which can be used to consume more resources than intended.",
        "cvssScore": 7.5,
        "cvssVector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
        "reference": "https://ossindex.sonatype.org/vulnerability/7ccaaed0-205b-4382-a963-8a30a0b151b1?component-type=npm&component-name=yargs-parser&utm_source=auditjs&utm_medium=integration&utm_content=4.0.1-beta.33"
      }
    ]
  }
]
```

**Comment by EMMLynch** [comment]
Closing since we've redesigned the site and functionality since this was created. If you feel that this is still needed, please let me know.
